Utilities/bootstrap

```diff
@@ -1012,6 +1032,11 @@ def main():
         build_flags.extend(["-Xswiftc", "-I{}".format(incdir)])
         build_flags.extend(["-Xswiftc", "-DHasCustomVersionString"])
     
+    if args.swift_build_dir:
+        swift_build_dir_search_path = os.path.join(args.swift_build_dir, "lib", "swift", args.sdk, args.arch)
+        build_flags.extend(["-Xlinker", "-L", "-Xlinker", swift_build_dir_search_path])
+        build_flags.extend(["-Xlinker", "-rpath", "-Xlinker", swift_build_dir_search_path])
```

Comment:
Hmm, I am not sure how correct is this. I think this will embed the absolute path of the bot as the rpath. We're already embedding another rpath at line 1018, we should probably keep the rpath related code together.

@ddunbar can you also review this patch?

Reply:
@aciidb0mb3r - yeah, I think that we should be using `-rpath-link` here, with the intention that the libraries will be laid out appropriately at runtime.